generators/client/index.js

```diff
@@ -315,6 +315,7 @@ module.exports = JhipsterClientGenerator.extend({
         writeSwaggerFiles: function () {
             // Swagger-ui for Jhipster
             this.template(MAIN_SRC_DIR + 'swagger-ui/_index.html', MAIN_SRC_DIR + 'swagger-ui/index.html', this, {});
+            this.template(MAIN_SRC_DIR + 'swagger-ui/_index-prod.html', MAIN_SRC_DIR + 'swagger-ui/index-prod.html', this, {});
```

Comment:
is this same as the `index.html` file or is there any difference? i didnt compare them